dynatrace-oneagent-operator-google-marketplace/schema.yaml

```diff
@@ -425,14 +402,7 @@ properties:
           - type: Role
             rulesType: CUSTOM
             rules:
-              - apiGroups:
-                  - policy
-                resources:
-                  - podsecuritypolicies
-                resourceNames:
-                  - dynatrace-oneagent-unprivileged
-                verbs:
-                  - use
```

Comment:
Empty rules are not for google marketplace

You have to remove all the service accounts

Reply:
But you have to add them in another way (prob. the deployer?), as the operator defaults to `dynatrace-oneagent-unprivileged` service account for the ds. We should prob. change that behaviour of the operator to set a default value to the sa for the ds.